update federation graphic

dthiex opened the pull request.

dthiex commented:
@jdries As I saw you updated this the other day. Should I remove CreoDias as "own" backend as it's more a different deployment location of your TerrasScope backend than it's own backend? Also are you okay with keeping the SH/EDC connection? Probably something I should also draw a arrow from our SH openEO backend to.

jdries commented:
indeed, best to remove creodias for now, gives us time to figure out what it will become
keeping the connection is fine, I was first thinking of making it to the sentinelhub backend, but if we keep edc as separate entity it should surely point to that!

dthiex commented:
Thanks, I removed creodias and moved EODC to the most left position as I think it's the nicest distribution.

*Technically the Sentinel Hub backend is our Sentinel Hub openEO backend which connects via EDC to Sentinel Hub but I think that will cause more confusion and in this graphic we don't need to be 100% accurate as it's more about the idea and which backends are involved.
